persistent/tests/test_timestamp.py

```diff
@@ -100,9 +100,10 @@ def test_ctor_from_string(self):
         self.assertEqual(ts.timeTime(), DELTA_SECS)
 
     def test_ctor_from_string_non_zero(self):
-        before = self._makeOne(2011, 2, 16, 14, 37, 22.0)
+        before = self._makeOne(2011, 2, 16, 14, 37, 22.80544)
         after = self._makeOne(before.raw())
         self.assertEqual(before.raw(), after.raw())
+        self.assertEqual(before.timeTime(), 1297867042.80544)
```

Comment:
Magic numbers (which can't be verified by eye) are problematic:  could we have generated the timestamp using the `time` module and the constants above?